src/mujincontrollerclient.cpp

```diff
@@ -167,7 +167,9 @@ void ObjectResource::GeometryResource::GetMesh(std::string& primitive, std::vect
 void ObjectResource::GeometryResource::SetGeometryFromRawSTL(const std::vector<unsigned char>& rawstldata, const std::string& unit, double timeout)
 {
     GETCONTROLLERIMPL();
-    MUJIN_ASSERT_OP_FORMAT(this->geomtype,!=,"mesh", "geomtype is not mesh: %s", this->geomtype, MEC_InvalidArguments);
+    if (this->geomtype != "mesh") {
```

Comment:
So this is behavior changing. Did the previous code have a bug?

Reply:
Yep, the previous code was buggy.

Reply:
I guess no one used this code before.

Reply:
I see, thanks for the fix.